src/TwbsHelper/Form/View/Helper/FormRow.php

```diff
@@ -35,12 +37,15 @@ public function __construct(\TwbsHelper\Options\ModuleOptions $moduleOptions)
      */
     public function render(\Laminas\Form\ElementInterface $element, ?string $labelPosition = null): string
     {
-
         // Nothing to do for hidden elements which have no messages
         if ('hidden' === $element->getAttribute('type') && !$element->getMessages()) {
             return parent::render($element, $labelPosition);
         }
 
+        if ($element instanceof LabelAwareInterface && $element->getLabelOption('label_position')) {
```

Comment:
This condition does not respect the current method call and its parameters. If a position is given, it is overwritten.